crates/sncast/src/starknet_commands/account/create.rs

```diff
@@ -181,8 +181,9 @@ async fn generate_account(
         }
         AccountType::Argent => {
             let factory =
-                ArgentAccountFactory::new(class_hash, chain_id, Felt::ZERO, signer, provider)
+                ArgentAccountFactory::new(class_hash, chain_id, Some(Felt::ZERO), signer, provider)
```

Comment:
This is for a guardian public key. I guess they changed the interface and now it should be `None`, but please check this 🙏 
```suggestion
                ArgentAccountFactory::new(class_hash, chain_id, None, signer, provider)
```

Reply:
Perhaps this is related to issues with argent accounts that you mentioned

Reply:
> This is for a guardian public key. I guess they changed the interface and now it should be None, but please check this 🙏

Yes, in `v0.4.0` guardian key is optional but none that I've used `Some(Felt::ZERO)` because we're still using class hash for v0.3.1. However, I think we're good to update to v0.4.1 (which is used while creating account) so the guardian key will be `None` then. Updated.